views/room.ejs

```diff
@@ -384,16 +385,31 @@
 
     <section class="tc-modal add-to-call-modal">
       <div class="tc-dialog">
-        <p class="title">Meeting link has been copied <span class="check-mark">&#10003;</span></p>
-        <p>Share link to add others to the meeting</p>
-        <input class="block" readonly id="current-url" onclick="this.select();">
-        <form id="dialOutForm">
-          <p class="title">Dial out</p>
-          <p>Enter the phone number:</p>
-          <input id="dialOutNumber" class="block" onclick="this.select();" 
-            placeholder="1-415-555-0123">
-          <button id="dialOutBtn" class="btn btn-blue btn-padding ctaarrow-white">Dial</button>
-        </form>
+        <header>
+          <i data-icon="close_gray_full" class="close"></i>
+          <div class="title" data-tab="feedback">
+            <span>Invite Users</span>
+          </div>
+        </header>
+        <secton class="dialog-content">
+          <div class="meeting-link">
+            <input class="current-url" readonly id="current-url" onclick="this.select();">
```

Comment:
Handlers have to be in the view instead of markup because this is generally considered bad practice.